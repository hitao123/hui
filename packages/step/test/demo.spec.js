import { renderToString } from '@vue/server-test-utils';
import stepDemo from '../demo/index.vue';

describe('Dialog component', () => {
  it('should show Button Demo', () => {
    expect(renderToString(stepDemo)).toMatchSnapshot();
  });
});
