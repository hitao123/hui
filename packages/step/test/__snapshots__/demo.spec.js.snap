// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog component should show Button Demo 1`] = `
<div class="demo-step">
  <h-steps active="2">
    <h-step>卖家下单</h-step>
    <h-step>商家接单</h-step>
    <h-step>买家提货</h-step>
    <h-step>交易完成</h-step>
  </h-steps>
  <div>
    <h-steps direction="vertical" active="1">
      <h-step>
        <h3>【城市】物流状态1</h3>
        <p>2016-07-12 12:40</p>
      </h-step>
      <h-step>
        <h3>【城市】物流状态1</h3>
        <p>2016-07-12 12:40</p>
      </h-step>
      <h-step>
        <h3>【城市】物流状态1</h3>
        <p>2016-07-12 12:40</p>
      </h-step>
      <h-step>
        <h3>【城市】物流状态1</h3>
        <p>2016-07-12 12:40</p>
      </h-step>
    </h-steps>
  </div>
</div>
`;
